web/libs/editor/src/components/CurrentEntity/AnnotationHistory.tsx

```diff
@@ -55,6 +55,8 @@ const DraftState: FC<{
 }> = observer(({ annotation, inline, isSelected }) => {
   const hasChanges = annotation.history.hasChanges;
   const store = annotation.list; // @todo weird name
+  const infoIsHidden = store.store.hasInterface("annotations:hide-info");
+  const hiddenUser = infoIsHidden ? { email: "Me" } : null;
```

Comment:
Is it possible that the user who will be replaced by "Me" is not actually the current one?

Reply:
Nwm. Looks like with `annotations:hide-info` we are going tto prevent such cases from appearing.